setup.py

```diff
@@ -8,9 +8,9 @@
     author='Edward FitzGerald',
```

Comment:
We should update these lines to be set to the team I think

Reply:
From a quick bit of reading, the recommended format is a list of comma separated authors, optionally with emails in <>s, and a single contact email address (which could be a mailing list). I'll leave this for you to change though, as you'll have a list of contributors.